project/plugins.sbt

```diff
@@ -6,3 +6,4 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.2")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.0")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.0")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```

Comment:
This dependency usually it's placed in the user's side

Reply:
Where should it be put?

Reply:
`~/.sbt/1.0/plugins/plugins.sbt`

Reply:
Removed in 671264eaa60f1103c3c15a565a8c0aacd4b3ab3a